### PR TITLE
[17.03] Fix issue backporting mount spec to pre-1.13 obj

### PR DESCRIPTION
**Pull request by cpuguy83**
Cherry-picks #32821 to 17.03 for 17.03.2

ping @thaJeztah @mlaventure

### CODE REVIEW

File: daemon/volumes.go [hunk 1]
```diff
@@ -243,48 +245,124 @@ func (daemon *Daemon) lazyInitializeVolume(containerID string, m *volume.MountPo
...
+		}
+		binds[mp.Destination] = mp
+	}
+
+	volumesFrom := make(map[string]volume.MountPoint)
+	for _, fromSpec := range container.HostConfig.VolumesFrom {
+		from, _, err := volume.ParseVolumesFrom(fromSpec)
+		if err != nil {
+			logrus.WithError(err).WithField("id", container.ID).Error("Error reading volumes-from spec during mount spec backport")
+		}
```
Comment (thaJeztah) [inline daemon/volumes.go]: May have missed this in the original PR as well, need a `continue` here 😓

**Review by thaJeztah — CHANGES_REQUESTED** [review]
noticed one thing we overlooked, but otherwise looks good

File: daemon/volumes.go (hunk 1 above)
Comment (cpuguy83) [inline daemon/volumes.go]: fixed

**Review by thaJeztah — APPROVED** [review]
LGTM

**Review by mlaventure — APPROVED** [review]
LGTM

**Comment by thaJeztah** [comment]
All green

**Pull request merged**